### PR TITLE
Update u301-url-shortener extension

### PR DESCRIPTION
**Pull request by shiny**
## Description
Now the extension supports custom domains.
Your own domain is available for the URL shortening.
If you'd rather use the default domain `u301.co`, keep the default value or leave the setting empty.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by pernielsentikaer** [comment]
Do you mind merging the conflicts before I can merge?

**Comment by shiny** [comment]
> Do you mind merging the conflicts before I can merge?

Conflicts resolved.

**Comment by shiny** [comment]
### Introduced The new feature

- Shorten URLs from Clipboard #13113
Screenshot 2024-06-24 at 13 56 01


### Bug Fixed
- set http header `Accept` to `application/json`

File: extensions/u301-url-shortener/package.json [hunk 1]
```diff
@@ -11,32 +11,21 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
-      "title": "Shorten URL",
+      "name": "shorten-clipboard",
```
Comment (pernielsentikaer) [inline extensions/u301-url-shortener/package.json:14]: Can one of the two commands keep the same `name`. It's the unique ID of the command where ranking, aliases and hotkeys are saved

File: extensions/u301-url-shortener/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "shorten-clipboard",
```
Comment (shiny) [inline extensions/u301-url-shortener/package.json]: is this correct?

File: extensions/u301-url-shortener/package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
...
+      "name": "index",
```
Comment (shiny) [inline extensions/u301-url-shortener/package.json:14]: is this correct?

Comment (pernielsentikaer) [reply extensions/u301-url-shortener/package.json:14]: Yes, since users might have an alias or hotkey assigned and it's bound by the `name` field

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**